docs/user-guide/07-partitioning.md

```diff
@@ -0,0 +1,55 @@
+# Partitioning
+
+The final partition table of an image built with image builder is determined by a combination of the following:
+1. The base partition table for a given image type.
```

Comment:
Does it make sense to link `images` here somehow? maybe not the main README as this does not contain any information :-/

Reply:
This information is only available in code and I wouldn't link to sources from a user guide.  The right thing to do would be to have a page with the default partition table for each image type.  I was thinking we could automate it—we can serialize the default PT for each distro + arch + image type and create a simplified description which automatically gets updated in the docs—but that's for another time.